engine/cmd/run.go

```diff
@@ -81,11 +82,14 @@ func run(ctx context.Context, c *config.Config, ns string, lv int) error {
 	}
 
 	mClient := metrics.NewClient(c.Autoscaler.MetricsWindow)
-	scaler := autoscaler.NewMultiAutoscaler(mgr.GetClient(), mClient, c.Autoscaler)
+	var scaler runtime.ScalerRegisterer
 	if c.Autoscaler.Enable {
+		scaler := autoscaler.NewMultiAutoscaler(mgr.GetClient(), mClient, c.Autoscaler)
```

Comment:
seems autoscaler inserted into the local val

Reply:
my bad! let me fix.